beater/config.go

```diff
@@ -66,23 +65,23 @@ func (s *Sourcemapping) isSetup() bool {
 	return s != nil && (s.esConfig != nil)
 }
 
-func (c *FrontendConfig) SmapAccessor() utility.SmapAccessor {
+func (c *FrontendConfig) SmapMapper() sourcemap.Mapper {
 	smap := c.Sourcemapping
 	if c.isEnabled() && smap.isSetup() {
-		if smap.accessor == nil {
-			smapConfig := utility.SmapConfig{
+		if smap.mapper == nil {
+			smapConfig := sourcemap.Config{
 				CacheExpiration:      smap.Cache.Expiration,
 				CacheCleanupInterval: smap.Cache.CleanupInterval,
 				ElasticsearchConfig:  smap.esConfig,
-				Index:                smap.Index,
+				Index:                smap.Index + "*",
 			}
-			smapAccessor, err := utility.NewSourcemapAccessor(smapConfig)
+			smapMapper, err := sourcemap.NewSmapMapper(smapConfig)
 			if err != nil {
-				logp.Err(fmt.Sprintf("Error creating Sourcemap Accessor: %v", err.Error()))
+				logp.Err(err.Error())
```

Comment:
Didn't consider this before but perhaps this should return an error instead?  That way the handler could do the logging instead of the library